src/EditorFeatures/Core.Wpf/Adornments/BrushTag.cs

```diff
@@ -0,0 +1,28 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Windows.Media;
+using Microsoft.VisualStudio.Text.Classification;
+using Microsoft.VisualStudio.Text.Editor;
+using Microsoft.VisualStudio.Text.Tagging;
+
+namespace Microsoft.CodeAnalysis.Editor.Implementation.Adornments
+{
+    internal abstract class BrushTag : ITag
```

Comment:
extracted out from GraphicsTag.  Used as the new feature wants a tag with a brush, but it doesn't want to handle drawing in the same fashion as the existing features.

Reply:
Would making an XML comment for this comparison to GraphicsTag be good, in case this comment is relevant to people besides those who saw the GitHub PR?

Reply:
sure!